.vib/argo-cd/goss/vars.yaml

```
@@ -0,0 +1,17 @@
+binaries:
+  - argocd
+directories:
+  - mode: "0775"
+    paths:
+    - /.argocd
+    - /app/config/gpg/keys
+    - /app/config/gpg/source
+    - /bitnami/argocd
+root_dir: /opt/bitnami
+version:
+  bin_name: argocd
+  flag: version
+sed_in_place:
+  exclude_paths:
+    # Ignore hack directory (not bitnami-related)
+    - \/opt\/bitnami\/argo-cd\/hack\/.*
```

I think this could also be defined as:

```suggestion
    - /opt/bitnami/argo-cd/hack
```